atom/browser/web_view_guest_delegate.cc

```diff
@@ -121,16 +104,9 @@ void WebViewGuestDelegate::HandleKeyboardEvent(
 }
 
 void WebViewGuestDelegate::RenderViewReady() {
-  // We don't want to accidentally set the opacity of an interstitial page.
-  // WebContents::GetRenderWidgetHostView will return the RWHV of an
-  // interstitial page if one is showing at this time. We only want opacity
-  // to apply to web pages.
-  auto render_view_host_view =
-      web_contents()->GetRenderViewHost()->GetWidget()->GetView();
-  if (guest_opaque_)
-    render_view_host_view->SetBackgroundColorToDefault();
-  else
-    render_view_host_view->SetBackgroundColor(SK_ColorTRANSPARENT);
+  // Set default UA-dependent background as transparent.
+  api_web_contents_->Send(new AtomViewMsg_SetTransparentBackground(
```

Comment:
I think we can just call this in `AtomRendererClient::RenderViewCreated` instead of sending a message.